packages/fast-color-explorer/app/control-pane.tsx

```diff
@@ -0,0 +1,381 @@
+                            height: "24px !important",
+                            "& div": {
+                                borderRadius: format(
+                                    "{0} !important",
+                                    toPx(cornerRadius)
+                                )(designSystem),
+                            },
+                        },
+                    },
+                },
+            },
+            ".sketch-picker input": {
+                boxShadow: "none !important",
+                background: neutralFillInputRest(designSystem),
+                color: neutralForegroundRest(designSystem),
+                border: `1px solid ${neutralOutlineRest(designSystem)} !important`,
+                height: height()(designSystem),
+                fontSize: "14px !important",
+                borderRadius: format("{0} !important", toPx(cornerRadius))(designSystem),
+                paddingTop: "0 !important",
+                paddingBottom: "0 !important",
+                "&:focus": {
+                    outline: "none",
+                    boxShadow: `0 0 0 2px ${neutralFocus(designSystem)} inset !important`,
+                },
+            },
+        },
+        controlPane: {
+            position: "relative",
+            zIndex: "1",
+            padding: "12px",
+            boxSizing: "border-box",
+            color: neutralForegroundRest,
+            height: "100%",
+            maxWidth: "300px",
+            boxShadow: (config: ColorsDesignSystem): string => {
+                return (elevation(ElevationMultiplier.e7)(undefined as any) as any)
+                    .boxShadow;
+            },
+            background: backgroundColor,
+        },
+    };
+};
+
+class ControlPaneBase extends React.Component<ControlPaneProps, ControlPaneState> {
+    private labelStyles: React.CSSProperties = {
+        marginBottom: "8px",
+    };
+
+    private inputStyles: React.CSSProperties = {
+        marginBottom: "18px",
+        width: "100%",
+    };
+    constructor(props: ControlPaneProps) {
+        super(props);
+        const defaultAccentBase: ColorRGBA64 | null = parseColorHexRGB(
+            this.props.designSystem.accentPalette[
+                Math.floor(this.props.designSystem.accentPalette.length) / 2
+            ]
+        );
+
+        const defaultNeutralBase: ColorRGBA64 | null = parseColorHexRGB(
+            this.props.designSystem.neutralPalette[
+                Math.floor(this.props.designSystem.neutralPalette.length) / 2
+            ]
+        );
+
+        this.state = {
+            accentColorBase:
+                defaultAccentBase instanceof ColorRGBA64
+                    ? defaultAccentBase.toStringHexRGB()
+                    : AccentColors.blue,
+            neutralColorBase:
+                defaultNeutralBase instanceof ColorRGBA64
+                    ? defaultNeutralBase.toStringHexRGB()
+                    : "#808080",

```

Comment:
seems like this value is captured in your neutralShortcuts, should you use that instead?